### ISSUE TITLE
Synchronization failed: unknown error

### ISSUE THREAD

**Issue by reuben-bradley** [michaelkourlas/voipms-sms-client#174]
Title is error message received when the app tries to sync my sms messages.

Account is set up with API password, and app correctly sets the callback URI. Have tried changing various synchronization settings, all to no avail. Any settings I may have overlooked, or other troubleshooting steps that you be suggested?

**Comment by michaelkourlas** [comment]
Unfortunately I can't say why you're experiencing that issue based on the information you've provided.

**Comment by reuben-bradley** [comment]
Appears to happen if I set the synchronization start date - I tried clearing the app data, set the account information and selected the phone number, and it properly synced the message received today (but not messages received earlier this month).  When I attempted to set the sync start date to August 1st, or even today, it began reporting the given error.  Only way to clear this field is to then wipe the app data once more.

Happy to provide any other information that may assist in troubleshooting this issue, just ask!
